swift/apple/FirezoneKit/Sources/FirezoneKit/Views/ResourceView.swift

```diff
@@ -68,6 +69,32 @@ struct ResourceView: View {
               }
           }
         }
+
+        if(model.favorites.ids.contains(resource.id)) {
+          Button(action: {
+            print("Removing favorite...")
```

Comment:
Are these print messages intended to be left in as debug statements? You'll want to use Log.d() instead if so so that they get flushed to the log file.

Reply:
Whoops forgot to clean those up